jscomp/test/ocaml_re_test.js

```diff
@@ -1081,7 +1083,7 @@ function split_at_match_rec(_l$p, _x) {
         case "TMatch" :
             return [
               List.rev(l$p),
-              Curry._1(remove_matches, x.tl)
+              remove_matches(x.tl)
```

Comment:
Interesting effect of not making `List.filter` implicitly curried. The use of `Curry` disappears even when compiled in curried mode.